### DISCUSSION TITLE
What 🥤 was originally called "Brad's Drink"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#885] [Galaxy Trivia] [answered]
**Bonus:** What does the current name stand for?

**Comment by peterbaccaro** [comment, marked as answer]
The drink originally called "Brad's Drink" is now known as **Pepsi-Cola**. It was created by Caleb Bradham, a pharmacist from North Carolina, United States, in the late 1890s. Bradham initially developed the carbonated beverage as a digestive aid and energy booster, combining carbonated water, sugar, vanilla, oils, pepsin, and cola nuts. He named the drink "Brad's Drink" after himself.

In 1898, Bradham officially changed the name of the beverage to Pepsi-Cola, **derived from the words "pepsin" and "cola"**. The new **name reflected the digestive and cola nut ingredients used in the formulation**. Over time, Pepsi-Cola gained popularity and became one of the major players in the soft drink industry, eventually shortening its name to Pepsi.
